src/UI/Art/SpriteArtist/PlayerSetupCommanderArtist.java

```diff
@@ -44,14 +46,33 @@ public static void draw(Graphics g, IController controller, ArrayList<CommanderI
     BufferedImage tooltip = SpriteUIUtils.makeTextFrame("Press Q for more info", 3, 2);
     myG.drawImage(tooltip, myWidth - tooltip.getWidth(), 3, null);
 
+    /////////////// Tag Picker Panels //////////////////////
+    if( myControl.shouldSelectMultiCO || myControl.tagCmdrList.size() > 1 )
```

Comment:
Uhhh, is it possible for `tagCmdrList.size()` to be greater than 1 if `!myControl.shouldSelectMultiCO`? Seems like just the first check should be sufficient (and the latter check an error case, if anything, assuming the first passes).

Reply:
Yes. You can get into that state via saved previous player configs for the map.
You could argue we should also save game rules for each map, but the player can just change those before going to the player setup screen anyway.
It is an error case, for sure.

Reply:
Oh. Yeah. If you start a tag game (so it saves the settings), then go to start a new game on that map without tags, it still shows the tag teams until you go into the CO-select screen. In fact, it shows the tags there until you back out, and then it fixes itself. That seems worth fixing.